sdk/python/kfp/components/structures.py

```diff
@@ -38,39 +37,21 @@
 
 
 @dataclasses.dataclass
-class InputSpec_:
```

Comment:
Core changes are here. The rest is to update tests and usage in response to this refactoring.